src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js

```diff
@@ -235,4 +235,28 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
     isHeavyOrSuper() {
         return this.weightclass === HEAVY_LETTER || this.weightclass === SUPER_LETTER;
     }
+
+    /**
+     * Returns the minimal distance that is required to a previous aircraft before another aircraft is allowed to use the runway.
+     *
+     * @for AircraftTypeDefinitionModel
+     * @method calculateSameRunwaySeparationDistanceInFeet
+     * @param previousTypeModel {AircraftTypeDefinitionModel} the aircraft type that used the runway before us.
+     * @returns {number} distance in feet
+     */
+    calculateSameRunwaySeparationDistanceInFeet(previousTypeModel) {
+        if (typeof previousTypeModel.category.srs === 'undefined'
+            || previousTypeModel.category.srs === 3) {
+            return 6000;
+        }
+
+        switch (this.category.srs) {
+            case 1:
+                return 3000;
+            case 2:
+                return 4500;
+            default:
+                return 6000;
```

Comment:
I agree with Nate, the 3000, 4500, and 6000 could be moved to `AIRPORT_CONSTANTS`, where the other similar ones are (or else to a new file for `SEPARATION_CONSTANTS` or something).
```
const AIRPORT_CONSTANTS = {
    ...,
    SRS_REDUCED_MINIMA: {
        CAT_1_IN_BACK: 3000,
        CAT_2_IN_BACK: 4500,
        CAT_3_INVOLVED: 6000
    },
    ...
}
```

Reply:
done